actioncable/test/client_test.rb

```diff
@@ -311,4 +342,12 @@ def test_server_restart
       assert_predicate c, :closed?
     end
   end
+
+  private
+    def stub_connection
```

Comment:
Maybe we can move this into `setup` (where we create a new server instance for each test)? Or set `config.connection_class` right in the example?

Reply:
👍 Moving this into the `setup` showed an error in the `test_unsubscribe_client` test where the channel identifier was being passed in instead of a connection identifier - I've fixed that